src/messages.ts

```diff
@@ -5,7 +5,11 @@ export const MESSAGE = {
     cantEstablishProxyConnection:     'Failed to connect to the proxy host at <a href="{url}">{url}</a>.',
     cantAuthorizeToProxy:             'Failed to authorize to the proxy at <a href="{url}">{url}</a>.',
     destRequestTimeout:               'Failed to complete a request to <a href="{url}">{url}</a> within the timeout period. The problem may be related to local machine\'s network or firewall settings, server outage, or network problems that make the server inaccessible.',
-    cantReadFile:                     'Failed to read a file at <a href="{url}">{url}</a> because of the error:\n\n{message}'
+    cantReadFile:                     'Failed to read a file at <a href="{url}">{url}</a> because of the error:\n\n{message}',
+    nodeError: {
+        'HPE_HEADER_OVERFLOW': 'The request header\'s size exceeds the set limit.\nIt causes an internal Node.js error on parsing this header.\nTo fix the problem, you need to specify the maximum header size via the NODE_OPTIONS=\'--max-http-header-size=...\' environment variable.',
```

Comment:
I think we need to add examples for programmatic and command-line interfaces for both error messages.
Also, if it's possible, try to retrieve the problematic header name from the error and use it in the error message.